lib/src/controls/yaru_title_bar.dart

```diff
@@ -207,7 +207,7 @@ class YaruTitleBar extends StatelessWidget implements PreferredSizeWidget {
               Row(
                 mainAxisSize: MainAxisSize.min,
                 children: [
-                  if (trailing != null) trailing!,
+                  ...?actions,
```

Comment:
this shortcut is so cool, didn't know this :heart_eyes_cat: 
How I can I ever love Java again :sob: